borgmatic/hooks/postgresql.py

```diff
@@ -202,13 +207,21 @@ def restore_database_dump(database_config, log_prefix, location_config, dry_run,
 
     If the extract process is None, then restore the dump from the filesystem rather than from an
     extract stream.
+
+    Use the given connection parameters to connect to the database. The connection parameters are
+    hostname, port, username, and password.
```

Comment:
It occurs to me you could use a `collections.namedtuple` for this data structure, just to formalize its contents a bit. Do not feel strongly at all.